properties/P000209.md

```diff
@@ -0,0 +1,6 @@
+---
+uid: P000209
+name: Density $\leq\mathfrak c$
+---
+
+The cardinality of a dense subset is $\leq \mathfrak c=|\mathbb R|=|2^\omega|$.
```

Comment:
```suggestion
There exists a dense subset with cardinality $\leq \mathfrak c=|\mathbb R|=|2^\omega|$.
```

Otherwise, it seems to be saying every dense subset has cardinality bounded by continuum, which is too strong.